sound/soc/sof/trace.c

```diff
@@ -13,10 +13,9 @@
 #include "sof-priv.h"
```

Comment:
Commit message:

ASoC: SOF: trace: remove code duplication in sof_wait_trace_avail()

Move duplicated code in sof_wait_trace_avail() to a helper function

Reply:
@plbossart Ack, fixed.